mypy/types.py

```diff
@@ -272,9 +272,12 @@ def serialize(self) -> JsonDict:
 class AnyType(Type):
     """The type 'Any'."""
 
-    def __init__(self, implicit: bool = False, line: int = -1, column: int = -1) -> None:
+    def __init__(self, implicit: bool = False, from_silent_import: bool = False,
+                 line: int = -1, column: int = -1) -> None:
```

Comment:
Nitpick: if you have to wrap a function def line, put each argument on a line of its own.  There's a lot of existing code in mypy which doesn't do this, but it's much easier to read if done this way, so we're trying to do this for new code.